pkgs/development/tools/rye/default.nix

```diff
@@ -35,12 +34,13 @@ rustPlatform.buildRustPackage rec {
   nativeBuildInputs = [ pkg-config ];
 
   buildInputs = [
-    git
     openssl
   ]
   ++ lib.optional stdenv.isDarwin SystemConfiguration;
```

Comment:
```suggestion
  ++ lib.optionals stdenv.isDarwin [
    Libsystem
    SystemConfiguration
  ];
```

Reply:
I am not really fluent with the darwin-specific changes. Are you satisfied with the current state ?

Reply:
Yes, I'm just waiting for ofborg's eval.